src/amb_kf.c

```diff
@@ -208,18 +314,35 @@ static void make_residual_measurements(nkf_t *kf, double *measurements, double *
   }
 }
 
+/** The temporal update step of the KF.
```

Comment:
Maybe add a bit more high level explanation of what this is doing? Description and name don't really match and may be confused with `update_kf_state`. Explain that this is the update for the _covariance_ specifically.